web/ui/src/Services/ServiceDetailsController.js

```diff
@@ -684,6 +684,40 @@
             }
         };
 
+        $scope.getServiceLogURL = function(service){
+            if(!service){
+                return "";
+            }
+            let a = LogSearch.getDefaultAppConfig(),
```

Comment:
Can you make these variables more descriptive?
Also, add comments for what this is doing